stdlib/docs/main.md

```diff
@@ -0,0 +1,6 @@
+- [std::crypto::hashes::blake3](./blake3_hashes.md)
+- [std::crypto::hashes::keccak256](./keccak256_hashes.md)
+- [std::crypto::hashes::sha256](./sha256_hashes.md)
+- [std::math::u256](./u256_math.md)
+- [std::math::u64](./u64_math.md)
+- [std::sys](./sys_std.md)
```

Comment:
instead of having `main.md` here, I'd probably integrate these into the `stdlib/README.md` file.

Reply:
It's easier to have a separate index file like this and we can include this file from stdlib/README.md.

Otherwise we will need to make README.md a template like what I did earlier, which I think it's probably unnecessary.

That sounds good to you?

Reply:
I think it is OK to keep `README.md` as a static file which we can update manually (I don't anticipate us adding new modules very frequently). Basically, I don't think we need to generate the index file dynamically - we can just have a readme which links back to individual module files and update it manually when we add/remove modules.

Reply:
fixed